### DISCUSSION TITLE
see the over hotend temperature on main screen (t0,t1, etc...)

### DISCUSSION THREAD

**Discussion by salocain** [KlipperScreen/KlipperScreen#501] [Ideas]
hello,

i don't see this option, but it's possible to see a t° of the second (third,etc ...) hotend on a main screen ?

thank you

**Comment by alfrix** [comment]
multiple hotend temps should show on main and temp, but only the active is shown in the status bar and job_status (printing)
the ability to see more than one extruder in those places has been requested in #668
